axe.d.ts

```diff
@@ -333,6 +332,14 @@ declare namespace axe {
     xpath: string[];
     ancestry: UnlabelledFrameSelector;
   }
+  interface DqElement extends axe.SerialDqElement {
+    element: Element;
+    toJSON(): axe.SerialDqElement;
```

Comment:
Do these need to reference the namespace, or should this just be `SerialDqElement`?